website/content/plugins/index.mdx

```diff
@@ -0,0 +1,7 @@
+---
+description: |
+  Plugins FIXME
+page_title: Plugins FIXME
+---
+
+# Plugins FIX ME
```

Comment:
💖 Love that this is stubbed out

Reply:
@azr would you be able to add some text here? Or tag someone else? 🙏

Reply:
Yes !